lib/routes/charge/pos_invoice.dart

```diff
@@ -233,31 +233,59 @@ class POSInvoiceState extends State<POSInvoice> {
+                                                                  bottom: 8.0,
+                                                                  right: 4.0,
+                                                                  top: 20.0),
+                                                          child: Image.asset(
+                                                            "src/icon/cart.png",
+                                                            width: 24.0,
+                                                            color: Theme.of(
+                                                                    context)
+                                                                .primaryTextTheme
+                                                                .button
+                                                                .color,
+                                                          ),
+                                                        ),
                                                         number: currentSale
                                                             .saleLines.length,
-                                                        onPress: () {
-                                                          var currentSaleRoute =
-                                                              CupertinoPageRoute(
-                                                                  fullscreenDialog:
-                                                                      true,
-                                                                  builder: (_) =>
-                                                                      SaleView(
-                                                                        useFiat:
-                                                                            _useFiat,
-                                                                      ));
-                                                          // FadeInRoute(
-                                                          //     builder: (_) =>
-                                                          //         SaleView(
-                                                          //           useFiat:
-                                                          //               _useFiat,
-                                                          //         ));
-                                                          Navigator.of(context)
-                                                              .push(
-                                                                  currentSaleRoute);
-                                                        },
-                                                        iconData: Icons
-                                                            .shopping_cart))),
+                                                        onPress: () {},
```

Comment:
This property is not being utilized on the BadgeIcon and should be removed.

Reply:
Done